src/lightning_utilities/core/imports.py

```diff
@@ -95,7 +95,7 @@ def _check_requirement(self) -> None:
                 self.message = f"Requirement {self.requirement!r} met"
             except Exception as ex:
                 self.available = False
-                self.message = f"Requirement {self.requirement!r} not met, {ex.__class__.__name__}: {ex}"
+                self.message = f"{ex.__class__.__name__}: {ex}. HINT: Try running `pip install {self.requirement!r}`"
```

Comment:
```suggestion
                self.message = f"{ex.__class__.__name__}: {ex}. HINT: Try running `pip install -U {self.requirement!r}`"
```

Should we include that in case it was a version mismatch?